src/card.css

```diff
@@ -0,0 +1,95 @@
+.card {
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding-top: 32px;
+  box-sizing: border-box;
```

Comment:
IMHO, it's better to use box-sizing: border-box for the universal selector(`*`)🤔.

b/c if we set it for all elements, we will get the signal behavior for the whole page)